src/game/server/entities/character.cpp

```diff
@@ -1363,9 +1363,9 @@ void CCharacter::FireWeapon()
 				if(GetClass() == PLAYERCLASS_SNIPER)
 				{
 					if(m_PositionLocked)
-						Damage = 30;
+						Damage = g_Config.m_InfSniperLockedLaserDmg; //30
 					else
-						Damage = random_int(10, 13);
+						Damage = g_Config.m_InfSniperLaserBaseDmg;
```

Comment:
you could add here Damage = g_Config.m_InfSniperLaserBaseDmg + random_int(0, 3);

Reply:
done